src/index.ts

```diff
@@ -103,6 +110,44 @@ async function main() {
     }
   }
 
+  async function configureApiKey(): Promise<string[] | false> {
```

Comment:
Can we name this more specifically? I have read this line several times and I have to look back to the description of the PR to remember what API it is for.

Reply:
Sure thing.

Reply:
I renamed it to `getApiKey` to mimic the existing accessors. My concern with this is that it has the side effect of `createApiKeyFile`, but it now more clearly describes its return value. I think this is probably fine.

Reply:
Well I mean, *what* API? `getAppStoreConnectApiKey` is what I was going for.

Reply:
Ah, gotcha. Addressed in latest push.